check-cf-space-quota-utilization/README.md

```diff
@@ -0,0 +1,116 @@
+# Receive notifications about Cloud Foundry resources utilization via SAP Alert Notification service
+
+Table of Contents
+
+* [Description](#description)
+* [Requirements](#requirements)
+* [Prerequisites](#prerequisites)
+* [How to use](#how-to-use)
+* [Expected result](#expected-result)
+
+## Description
+
+This command allows you to get insights about the resource utilization in your Cloud Foundry runtime, like memory consumption and service usage in your Cloud Foundry space and its parrent Cloud Foundry organization. 
+Forward the insights towards your [SAP Alert Notification service](https://help.sap.com/docs/alert-notification/sap-alert-notification-for-sap-btp/what-is-sap-alert-notification-service-for-sap-btp) account where you can subscribe for events regarding your resource consumption.
+By using an appropriate configuration you could get informed about detailed resource usage, and you can also be notified when your resource usage is close the quota defined for your space and organization as well.
+The event which will be triggered towards your SAP Alert Notification instance is described in the following [documentation](https://help.sap.com/docs/alert-notification/sap-alert-notification-for-sap-btp/resource-quota-utilization-events?locale=en-US#technical-details)
+
+## Requirements
+
+* A Cloud Foundry Runtime within SAP BTP.
+
+* An instance of [SAP Alert Notification service instance](https://help.sap.com/docs/alert-notification/sap-alert-notification-for-sap-btp/initial-setup) in the Cloud Foundry space which resource utilization metrics will be pulled.
+
+* A technical user with **Space Auditor** and **Organization Auditor** roles in Cloud Foundry space and the corresponding organization that will be used to access the resource data in the Cloud Foundry space. For further information about user management refer to the following [documentation](https://help.sap.com/docs/btp/sap-business-technology-platform/administration-and-operations-in-cloud-foundry-environment).
+
+## Prerequisites
+
+* Create an instance of SAP Alert Notification service and configure a subscription that will match the [Resource quota utilization event](https://help.sap.com/docs/alert-notification/sap-alert-notification-for-sap-btp/resource-quota-utilization-events?locale=en-US#technical-details).
+  You can directly import the following example Alert Notification service configuration that will forward the resulting event from the command execution to your email:
+  ```java
+    {
```

Comment:
Extract the exported configurations in a separate file and reference it here (similar to `catalog.json`)